forms-flow-admin/src/index.tsx

```diff
@@ -146,6 +146,23 @@ const Admin = React.memo(({ props }: any) => {
                 />
               )}
             />)}
+            <Route 
+            exact
+            path={`${BASE_ROUTE}admin`}
+             >
+              {
+                userRoles.length && (
+                  <Redirect 
+                    to ={
+                      isDashboardManager ? `${BASE_ROUTE}admin/dashboard` 
+                      :isRoleManager ? `${BASE_ROUTE}admin/roles`
+                      :isUserManager ? `${BASE_ROUTE}admin/users`
```

Comment:
instead of redirect 404 can we render unauthorized page ?

Reply:
yes.updated